src/tokens.scss

```diff
@@ -0,0 +1,10 @@
+:host {
+  --ds-auro-dropdown-help-text-color: var(--ds-color-text-secondary-default, $ds-color-text-secondary-default);
+  --ds-auro-dropdown-label-text-color: var(--ds-color-text-secondary-default, $ds-color-text-secondary-default);
+  --ds-auro-dropdown-popover-background-color: var(--ds-color-container-primary-default, $ds-color-container-primary-default);
+  --ds-auro-dropdown-popover-border-color: transparent;
```

Comment:
I think in this case it needs to default to the same color as the container - otherwise it would look 1 pixel slimmer.

Reply:
The border in the currently released version is already set to `transparent` and I am not seeing a visual change when clicking on the dropdown trigger. We can discuss this further if you'd like.